wxc/wxc.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}/bin
+libdir=${prefix}/lib
+includedir=${prefix}/include/@PROJECT_NAME@-@PROJECT_VERSION_MAJOR@.@PROJECT_VERSION_MINOR@/@PROJECT_NAME@
```

Comment:
I think this is a bit nonstandard.

Since it's a "new" package we could do a breaking change and have a more common (I think) configuration:

*  `includedir=${prefix}/include`
* header files go under `${includedir}/wxc`, for example `${includedir}/wxc/wxc.h`
* in wxcore c includes, add the `wxc` directory: `#include "wxc/wxc.h"`

(but let's merge this for now)

Reply:
Yeah, I'd be keen to do things more conventionally.